posthog/api/capture.py

```diff
@@ -649,7 +667,12 @@ def get_event(request):
         )
 
     statsd.incr("posthog_cloud_raw_endpoint_success", tags={"endpoint": "capture"})
-    return cors_response(request, JsonResponse({"status": 1}))
+
+    response_body: dict[str, int | list[str]] = {"status": 1}
+    if recordings_were_quota_limited and settings.RECORDINGS_QUOTA_LIMITING_RESPONSES:
```

Comment:
Given the 429 responses caused an incident before we changed the retry logic clientside would it be worth adding a check here to only return the `quota_limited` response if requests come from library versions greater than the version we fixed the logic in?

Reply:
so, we're not returning 429 any more...

just returning the body, and the old libraries don't parse the body (I believe)

so i _think_ we're safe - that said I think we do now have the library version available in this code 🤔